decidim-elections/app/views/decidim/elections/votes/_onboarding_modal.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="reveal" data-reveal id="onboarding-modal">
+  <div class="reveal__header">
+    <h3 class="reveal__title"><%= t("decidim.elections.votes.onboarding_modal.title") %></h3>
+    <button class="close-button" data-close aria-label="Close modal" type="button">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+  <p>
+    <%= t("decidim.elections.votes.onboarding_modal.description") %>
+  </p>
+  <div class="row mt-sm">
+    <div class="columns medium-8 medium-offset-2">
+      <%= link_to decidim.new_user_registration_path, class: "button button--sc expanded" do %>
+        <%= t("decidim.elections.votes.onboarding_modal.create_account") %>
+      <% end %>
+      <div class="text-center">
+        <button class="link" data-close><%= t("decidim.elections.votes.onboarding_modal.no_account") %></button>
```

Comment:
👀 This button is missing the `type` attribute:

```suggestion
        <button class="link" data-close type="button"><%= t("decidim.elections.votes.onboarding_modal.no_account") %></button>
```

Reply:
I'll add it. Thanks!